### PR TITLE
Making errors a bit more reportable

### PR DESCRIPTION
**Pull request by jdivock**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jdivock has signed the CLA. Thanks for the pull request!

**Comment by j3tan** [comment]
lgtm

**Comment by nzakas** [comment]
Since there's no issue describing this change, can you summarize the need?

Also, can this be tested?

**Comment by jdivock** [comment]
Right now the t3 error wrapping just provides a string, rather that writing up a regex to break it up into the data I need I'd rather split it up in the error that's generated. This is related to another piece that's taking that metadata and sending it off to Raygun for proper error bucketing.

I'll work on a test today.

**Comment by nzakas** [comment]
OK. It's a good practice to open an issue for each PR, as that makes it easier for others to follow development, so I'd suggest doing that from now on.

One last bit of feedback: wouldn't you also want to know if it's a module, behavior, or service? The object and method alone, without that ts missing context.

**Comment by jdivock** [comment]
Tests added.

Feedback tracked in this issue https://github.com/box/t3js/issues/80

**Comment by nzakas** [comment]
+1

**Comment by j3tan** [comment]
lgtm
